sdk/personalizer/ci.yml

```diff
@@ -20,11 +20,10 @@ pr:
       - feature/v4
   paths:
     include:
-      - sdk/personalizer/ci.mgmt.yml
-      - sdk/personalizer/ai-personalizer/
```

Comment:
There is no folder with the name `ai-personalizer`.
But also, looks like we need to provide the service directory so that it gets the `ci.yml`?!

Reply:
do we want sdk/personalizer/ai-personalizer-rest?